config/webpack.plugins.js

```diff
@@ -28,6 +28,7 @@ const plugins = [
     shared: [
       { react: { singleton: true, requiredVersion: deps.react } },
       { 'react-dom': { singleton: true, requiredVersion: deps['react-dom'] } },
+      { 'react-redux': { singleton: true, requiredVersion: deps['react-redux'] } },
```

Comment:
Is this required? It can actually break the dependencies at runtime. Hooks will be referencing to incorrect store.

Reply:
Yup, inventory is using react-redux functions and without this it trhew an error with missing dependency. We could remove the singleton here

Reply:
Yeah we have to remove the singleton